addons/udes_stock/models/stock_picking.py

```diff
@@ -80,6 +80,12 @@ class StockPicking(models.Model):
         compute="_compute_related_picking_ids",
         help="Created Back Orders",
     )
+    u_original_picking_id = fields.Many2one(
+        "stock.picking",
+        string="Original Picking",
+        help="The original picking in the chain",
```

Comment:
I wouldn't consider it as original picking of the chain, since a chain can be considered a multi steps route. I would probably consider this field the original picking of the backorders.